templates/layout.html

```diff
@@ -110,25 +110,97 @@
+          z-index: 1;
+          padding-top: 100px;
+          left: 0;
+          top: 0;
+          width: 100%;
+          height: 100%;
+          overflow: auto;
+          background-color: rgb(0, 0, 0);
+          background-color: rgba(0, 0, 0, 0.4);
+        }
+
+        /* Modal Content */
+        .modal-content {
+          background-color: #fefefe;
+          margin: auto;
+          padding: 20px;
+          border: 1px solid #888;
+          width: 80%;
+        }
+
+        /* The Close Button */
+        .close {
+          color: #aaaaaa;
+          float: right;
+          font-size: 28px;
+          font-weight: bold;
+        }
+
+        .close:hover,
+        .close:focus {
+          color: #000;
+          text-decoration: none;
+          cursor: pointer;
+        }
+      </style>
+
+      <li class="nav-item dropdown dropdown-pull-right">
+        <a id="myBtn" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button"
+          aria-haspopup="true" aria-expanded="false">
           <i class="fa fa-globe" aria-hidden="true"></i> <span class="caret"></span>
         </a>
-        <div class="dropdown-menu" aria-labelledby="dropdown01">
-          {%- set eng_active = this.alt == 'en' %}
-          <a class="dropdown-item {% if eng_active %}active{% endif %}" href="{{ '.'|url(alt='en') }}">
-          English
-          </a>
-
-          {%- for alt, alternative in config.ALTERNATIVES.items()|sort -%}
-          {%- set alt_active = alt == this.alt -%}
-          {%- if alt != config.primary_alternative -%}
-          <a class="dropdown-item {% if alt_active %}active{% endif %}" href="{{ '.'|url(alt=alt) }}">
-          {{ transbag('translate', alt, alternative.name.en.lower()) }}
-          </a>
-          {%- endif -%}
-          {%- endfor -%}
+        <!-- Creating Modal Window Div -->
+        <div id="myModal" class="modal">
+          <!-- Modal Content Div [Language List Shows] -->
+          <div class="modal-content">
+            <span class="close">&times;</span>
+            {%- set eng_active = this.alt == 'en' %}
+            <a class="dropdown-item {% if eng_active %}active{% endif %}" href="{{ '.'|url(alt='en') }}">
+              English
+            </a>
+            {%- for alt, alternative in config.ALTERNATIVES.items()|sort -%}
+            {%- set alt_active = alt == this.alt -%}
+            {%- if alt != config.primary_alternative -%}
+            <a class="dropdown-item {% if alt_active %}active{% endif %}" href="{{ '.'|url(alt=alt) }}">
+              {{ transbag('translate', alt, alternative.name.en.lower()) }}
+            </a>
+            {%- endif -%}
+            {%- endfor -%}
+            <div class="text-center mt-2">
+              <small>This modal designed by: <b><a style="text-decoration: none; color: rgb(38, 36, 36);" target="_blank" href="https://github.com/pythoniseasy-hub">Gagan Aggarwal,</a></b> </br> Email: <b>gaggarwal124@gmail.com</b></small>
```

Comment:
Um... no. A drive by fix to one feature of a website does not give you permission to add personal promotion links to our website.